ds/beis_indicators/defra/make_defra_indicators.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import numpy as np
+import os
+import pandas as pd
+from zipfile import ZipFile
+
+from beis_indicators.utils.nuts_utils import nuts_earliest
+from beis_indicators.defra.defra_processing import (coordinates_to_points, 
+        translate_coordinates)
+
+logger = logging.getLogger(__name__)
+
+def get_pollution_data(year, pollution_type):
+    '''get_pollution_data
+    Downloads and stores pollution data from https://uk-air.defra.gov.uk/data/pcm-data
+    
+    Args:
+        year (int): Year to collect data from. Check website for coverage.
+        pollution_type (str): Name of pollutant. Choices are currently pm10 
+            and pm25, no2 or nox.
+    '''
+    logger.info(f'Fetching {pollution_type} data from DEFRA')
+
+    base_url = 'https://uk-air.defra.gov.uk/datastore/pcm/map{}.csv'
+        
+    raw_data_dir = f'{project_dir}/data/raw/defra/'
```

Comment:
SHouldn't we import `project_dir`? Also it would be good to pass the dir as an argument of the function rather than using it as a global possibly